taskcluster/docker/linux/Dockerfile

```diff
@@ -103,9 +103,12 @@ RUN apt-get update -qq \
 
 RUN pip3 install --upgrade pip
 RUN pip3 install \
+    pytest \
     pyyaml \
     toml \
     taskcluster \
+    # For formatting uniffi generated python code
+    yapf \
```

Comment:
Does this get used anywhere? Fwiw, we tend to use `black` for most Python projects at Mozilla.

Reply:
Thanks for the explanation! Let's just leave it as is then.